.github/CODEOWNERS

```diff
@@ -36,31 +36,31 @@
 # In this example, @doctocat owns any file in the `/docs`
 # directory in the root of your repository.
 #/docs/ @doctocat
-/accounts/      @KimKyungup @aidan-kwon @jeongkyun-oh @yoomee1313 @hqjang-pepper
-/api/           @KimKyungup @aidan-kwon @jeongkyun-oh @yoomee1313 @jimni1222 @sirano11 @aeharvlee @mckim19
-/blockchain/    @KimKyungup @aidan-kwon @jeongkyun-oh @ehnuje @winnie-byun @jimni1222 @sirano11 @mckim19
-/build/         @KimKyungup @aidan-kwon @jeongkyun-oh
-/client/        @aidan-kwon @jeongkyun-oh @hqjang-pepper
-/cmd/           @KimKyungup @aidan-kwon @jeongkyun-oh @yoomee1313
+/accounts/
+/api/           @jimni1222 @sirano11 @aeharvlee
+/blockchain/    @KimKyungup @aidan-kwon @jeongkyun-oh @ehnuje
+/build/         @henry-will @hyunsooda @hqjang-pepper @nohkwak @2dvorak
```

Comment:
I don't know about this file and change history.
I think build directory's owner should include @KimKyungup @aidan-kwon @jeongkyun-oh.

Reply:
I re-arranged service chain developers. Please, take a look again.